File: Data Structures & Algorithms/top-k-elements-in-list/submission-0.java
class Solution {
    public int[] topKFrequent(int[] nums, int k) {
        HashMap<Integer,Integer> map=new HashMap<>();
        for(int n:nums){
            map.put(n,map.getOrDefault(n,0)+1);
        }
        PriorityQueue<Map.Entry<Integer, Integer>> max=new PriorityQueue<>((a,b)->b.getValue()-a.getValue());
        max.addAll(map.entrySet());
        int[] top=new int[k];
        for(int i=0;i<k&&!max.isEmpty();i++){
            top[i]=max.poll().getKey();
        }
        return top;
    }
}

File: Data Structures & Algorithms/divide-array-into-equal-pairs/submission-0.py
class Solution:
    def divideArray(self, nums: List[int]) -> bool:
        count={}
        for i in nums:
            if i in count:
                count[i]+=1
            else:
                count[i]=1
        for i in count.values():
            if i%2!=0:
                return False
        return True